gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,4 @@
-#Tue Jan 08 21:49:33 IST 2019
```

Comment:
remove this file from commit also

Reply:
this is done

Reply:
@gateway2rohan
remove this file from the commit.